src/java/picard/sam/AddOrReplaceReadGroups.java

```diff
@@ -30,13 +30,32 @@
  * @author mdepristo
  */
 @CommandLineProgramProperties(
-        usage = "Replaces all read groups in the INPUT file with a single new read group and assigns " +
-                "all reads to this read group in the OUTPUT BAM",
-        usageShort = "Replaces read groups in a BAM or SAM file with a single new read group",
+        usage = AddOrReplaceReadGroups.USAGE_SUMMARY + AddOrReplaceReadGroups.USAGE_DETAILS,
+        usageShort = AddOrReplaceReadGroups.USAGE_SUMMARY,
         programGroup = SamOrBam.class
 )
 public class AddOrReplaceReadGroups extends CommandLineProgram {
-
+    static final String USAGE_SUMMARY = "Replaces read groups in a BAM file.  ";
+    static final String USAGE_DETAILS = "This tool enables the user to merge all the read group INPUT files into a single " +
```

Comment:
Couple of words got lost here. I think this should still say "merge all the read groups *in the* INPUT files.